src/Tribe/Assets.php

```diff
@@ -602,16 +600,53 @@ public function remove( $slug ) {
 	 *
 	 * @since 4.3
 	 *
-	 * @param string $slug Slug of the Asset.
+	 * @param string|array $slug Slug of the Asset.
+	 * @param boolean      sort  If we should do any sorting before returning.
 	 *
 	 * @return array|object|null Array of asset objects, single asset object, or null if looking for a single asset but
 	 *                           it was not in the array of objects.
 	 */
-	public function get( $slug = null ) {
+	public function get( $slug = null, $sort = true ) {
 		if ( is_null( $slug ) ) {
+			if ( $sort ) {
+				$cache_key_count = __METHOD__ . ':count';
+				// Sorts by priority.
+				$cache_count = tribe_get_var( $cache_key_count, 0 );
+				$count = count( $this->assets );
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 7 spaces but found 1 space

<sub>hash: 474483721e29cda1566add67add35284</sub>